docs/admin/gatewayapi_migration.md

```diff
@@ -0,0 +1,245 @@
+# Kubernetes Ingress to Gateway API Migration Guide
+
+## 1. Install Gateway API CRD
+Install the Gateway API CRD as it is not part of the Kubernetes installation. KServe implements Gateway API version 1.2.1.
```

Comment:
The Gateway API is not part of the Kubernetes cluster, therefore it needs to be installed manually, to do this, follow the next steps.